aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -938,6 +1136,15 @@ class AztecText : EditText, TextWatcher {
         }
     }
 
+    fun getAppliedHeader(selectionStart: Int, selectionEnd: Int): TextFormat? {
+        TextFormat.values().forEach {
+            if (contains(it, selectionStart, selectionEnd)) {
```

Comment:
Will the output of this method change if we change the order of elements inside TextFormat and apply some other style to header?

Let's say the TextFormat will look like this:
```
enum class TextFormat {
    FORMAT_BOLD,
    FORMAT_ITALIC,
    FORMAT_UNDERLINED,
    FORMAT_STRIKETHROUGH,
    FORMAT_BULLET,
    FORMAT_QUOTE,
    FORMAT_LINK,
    FORMAT_NORMAL,
    FORMAT_HEADER_1,
    FORMAT_HEADER_2,
    FORMAT_HEADER_3,
    FORMAT_HEADER_4,
    FORMAT_HEADER_5,
    FORMAT_HEADER_6
}
```
Now, we apply bold to some text, and then apply header style to it. In this case getAppliedHeader will return FORMAT_BOLD, because it encountered it first.